css/app.css

```diff
@@ -215,6 +215,18 @@ html, body {
   padding-left: 3px;
 }
 
+#graphiql-container .tool-button {
+  cursor: pointer;
+  text-decoration: none;
+  font-family: sans-serif;
+  color: #ccc;
+  padding: 2px 4px;
+  margin: 0 4px;
+}
+#graphiql-container .tool-button:hover {
+  color: #333;
```

Comment:
Maybe more of a personal preference, but a lighter button feels like the query's not possible to be prettified to me (like a disabled button). Instead of :hover how about :active selector for the clickable feedback, just like .execute-button?